sourcetable_test.go

```diff
@@ -116,3 +118,74 @@ func TestSourcetableString(t *testing.T) {
 		t.Errorf("sourcetable did not convert to string correctly")
 	}
 }
+
+func TestDecodeSourcetable(t *testing.T) {
+
+	// Arrange
+	var (
+		table = `
+		CAS;auscors.ga.gov.au;2101;AUSCORS Ntrip Broadcaster;GA;0;AUS;-35.34;149.18
```

Comment:
This is what I was talking about the other day - in the BKG caster implementation the sourcetable is configured from a file and it's not checked, so some people (including ourselves) implement it incorrectly. 

Technically that CAS entry should have three more fields.

Reply:
Is it worth fixing this now?

Reply:
I think so. It's technically an invalid CasterEntry. Here's the IGS caster which is formatted correctly: http://igs-ip.net:2101/

I'll get onto sorting out the AUSCORS Sourcetable.